docs/modules/ROOT/pages/cops_rspec.adoc

```diff
@@ -2503,16 +2503,24 @@ end
 | Yes
 | No
 | 1.0
-| -
+| 2.4
 |===
 
-Checks for multiple top-level example groups.
+Checks for multiple top-level example groups or nested example groups.
+
+This cop can be configured using the `EnforcedStyle` option.
+
+Using the `EnforcedStyle: nesting`, multiple descriptions for the
+same class or module should either be nested or separated into different
+test files.
 
-Multiple descriptions for the same class or module should either
-be nested or separated into different test files.
+Using the `EnforcedStyle: splitting`, nested describe for the same class
```

Comment:
"describe" -> "example group". It's a shortcoming the the cop only operates on `describe`, and not e.g. `feature`.

Reply:
I understand. I corrected the sentence.